src/ui/buttons/styles/color.js

```diff
@@ -57,7 +57,7 @@ export const buttonColorStyle = `
     }
 
     .${ CLASS.BUTTON }[${ ATTRIBUTE.FUNDING_SOURCE }=${ FUNDING.VENMO }].${ CLASS.COLOR }-${ BUTTON_COLOR.BLUE } {
-        background: #009cde;
+        background: #008CFF;
```

Comment:
@gabrielo91 I noticed this color code `#009cde` is used 7 times in color.js (https://github.com/paypal/paypal-checkout-components/search?q=009cde). Do we need to update all 7 or just this one occurrence?

Reply:
We had a discussion about that and yes, the value `#009cde` matches several times, however, the only one that affects the Venmo button is in that line. The other occurrences are part of the Paypal button and its variants, for example, in the box-shadow color

![image](https://user-images.githubusercontent.com/17114924/119386543-4f31b380-bc8d-11eb-8de5-72450df99ef7.png)

Reply:
So yes, venmo and paypal buttons share that color, however, according to @mstuart we should only update the one that affects to it

Reply:
okay cool. Glad you hear you already considered this 👍